src/config.js

```diff
@@ -9,8 +9,8 @@ const config = {
   currencyConverterApi: {
     baseUrl: 'https://free.currconv.com/api/v7',
     key: 'ca0133cec785f144a8c5'
-  }
-}
+  },
+  itemsPerPage: 10
```

Comment:
Can you please make it configurable from an env variable too, similar to db creds above?